sdk/src/main/java/com/meniga/sdk/models/eventtracking/operators/MenigaEventTrackingOperationsImp.java

```diff
@@ -0,0 +1,18 @@
+package com.meniga.sdk.models.eventtracking.operators;
+
+import com.meniga.sdk.MenigaSDK;
+import com.meniga.sdk.helpers.Result;
+import com.meniga.sdk.models.eventtracking.MenigaEventTracking;
+import com.meniga.sdk.webservices.eventtracking.TrackEvent;
+
+/**
+ * Copyright 2018 Meniga Iceland Inc.
+ * Created by agustk on 31.5.2018.
+ */
+public class MenigaEventTrackingOperationsImp implements MenigaEventTrackingOperations {
+	@Override
```

Comment:
Purist comment mode ON - you could switch to spaces instead of tabs here.

Reply:
The project should be using tabs :p :)